swan-cern/.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 0.1.11
+commit = True
+tag = True
+tag_name = swan-cern@{new_version}
+message = Bump chart swan-cern/ version: {current_version} → {new_version}
+
+[bumpversion:file:Chart.yaml]
```

Comment:
As per the comment you made in the PR message, this looks for a string with {current_version} in Chart.yaml and replaces it with {new_version}, no matter where?

Reply:
Also, how is {new_version} filled? Does the value come from what the user selects in the UI (minor, major)?

Reply:
Yes, this is a simple find/replace, and can potentially conflict if any dependency has the same `current_version`, it gets bumped as well. (This is especially possible in our other extensions, in packages.json files particularly, I'm surprised we haven't had problems yet.)

The [bump2version](https://pypi.org/project/bump2version/) tool has a `search=default:{current_version}` option which can be customized with a python template that matches only the areas we want in the file.

The `new_version` is calculated from the 2nd argument to bump2version command, `part` which is major/minor/patch/'other parts in SemVer', and is currently an input to the workflow in this pipeline

Reply:
Ok thanks, but how and when and by whom is the version we are bumping to specified? Is it the same as with the extensions, that I select on the UI which version (major, minor, patch) I want to bump to? Does that setting apply to both swan and swan-cern?

Reply:
Currently when triggering the workflow manually, the part to bump is an input, with a drop down to select  'major', 'minor', and 'patch'.

This applies to both `swan/` and `swan-cern/`. If you select major, both charts will be tagged with a new major version.

We can also provide two inputs for each chart separately, what do you suggest?

Reply:
It might be the case that the changes for swan deserve a major version change, but not for swan-cern. Moreover, since we agreed both charts can have different versions, I believe it would be better to have two variables, one for each.

Reply:
I have updated it now to include two dropdowns to pick major/minor/patch for each